docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -64,8 +64,18 @@ desktop icon in the first column to distinguish them from SSH recordings.
 
 ![Desktop Session Recording](../../../img/desktop-access/session-recording@2x.png)
 
-Click the play button to open the player in a new tab. To export desktop session
-recordings to video for playback outside of Teleport, use the
+Click the play button to open the player in a new tab.
```

Comment:
Let's just remove the whole snippet about `tsh recordings export`.

Reply:
Removed @zmb3 , take out this too? https://goteleport.com/docs/reference/cli/tsh/#tsh-recordings-export

Reply:
Sure

Reply:
The reference to tsh recordings export has been remove, please approve after reviewing.